README.markdown

```diff
@@ -396,19 +396,29 @@ SSL handshake if the `wss://` scheme is used.
 
 * `client_cert`
 
-    Specifies a client certificate chain cdata object that will be used while TLS handshaking with remote server. 
-    These objects can be created using 
-    [ngx.ssl.parse_pem_cert](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_cert) 
-    function provided by lua-resty-core. 
+    Specifies a client certificate chain cdata object that will be used while TLS handshaking with remote server.
+    These objects can be created using
+    [ngx.ssl.parse_pem_cert](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_cert)
+    function provided by lua-resty-core.
     Note that specifying the `client_cert` option requires corresponding `client_priv_key` be provided too. See below.
 
 * `client_priv_key`
 
-    Specifies a private key corresponds to the `client_cert` option above. 
-    These objects can be created using 
-    [ngx.ssl.parse_pem_priv_key](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_priv_key) 
```

Comment:
In cases like this our goal is to maintain a minimal footprint on the upstream codebase; these changes are not absolutely necessary to this patch, and should be avoided.